drivers/amazonec2/amazonec2.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/docker/machine/drivers/util"
```

Comment:
It's not idiomatic go to have a pkg named util. Call it driverutil or something pls. :-)